src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,18 +11,18 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private String feedbackToUser;
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
     public CommandResult(String feedbackToUser) {
-        this.feedbackToUser = feedbackToUser;
+        this.setFeedbackToUser(feedbackToUser);
```

Comment:
Note that method of a class can directly access its fields regardless of visibility (i.e. private, protected, public members are all accessible to the method of the same class).